integration/run_test

```diff
@@ -4,8 +4,6 @@ set -e
 
 source ./integration/common
```

Comment:
Should the path be `./common`? As is already inside `./integration` dir

Reply:
It's assumed that this script is run from the root directory of this repo, which should make the correct path `./integration/common`